polymetis/polymetis/python/torchcontrol/models/torchscript_pinocchio.py

```diff
@@ -47,21 +81,41 @@ def get_joint_velocity_limits(self) -> torch.Tensor:
         return self.model.get_joint_velocity_limits()
 
     def forward_kinematics(
-        self, joint_positions: torch.Tensor
+        self,
+        joint_positions: torch.Tensor,
+        link_name: str = "",
```

Comment:
Maybe preferable to use None here rather than empty string?  Conceivably, "" could maybe be a valid link name in some particularly unfriendly URDF?